src/print_sum_of_all_fields.md

```diff
@@ -0,0 +1,52 @@
+
+# Print the sum of all fields on each line
+
+*Pomodoro: 2*
+
+This one-liner turns on field auto-splitting with the -a command-line option and imports the sum function from the List::Util module with -Mlist::Util=sum.
+
+```perl
+perl -MList::Util=sum -alne 'print sum @F'
+```
+
+For example, the line 1 4 8 would be split on each space so that @F would become (1, 4, 8). The sum @F statement sums the elements in the @F array, giving you 13.
```

Comment:
```suggestion
For example, the line 1 4 8 would be split on each space so that @F would become `(1, 4, 8)`. The sum `@F` statement sums the elements in the `@F` array, giving you 13.
```

Reply:
And this is how the `suggestion` feature on Github works.
https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-a-suggested-change